openapi/mx_platform_api.yml

```diff
@@ -1512,32 +1512,49 @@ components:
       type: object
     InstitutionResponse:
       properties:
+        created_at:
+          example: '2016-10-13T18:07:57.000Z'
+          nullable: true
```

Comment:
I do not think this nullable, correct me if I am wrong but won't `created_at`  always be populated?